cli/src/main/kotlin/commands/ReporterCommand.kt

```diff
@@ -207,7 +207,9 @@ class ReporterCommand : OrtCommand(
             ortResult = ortResult.replaceConfig(config)
         }
 
-        val resolutionProvider = DefaultResolutionProvider.create(ortResult, resolutionsFile)
+        val resolutionProvider = ortResult.resolvedConfiguration.resolutions?.let { resolutions ->
+            DefaultResolutionProvider().add(resolutions)
```

Comment:
Unrelated, but this made me wonder why we don't have a `DefaultResolutionProvider(resolutions)` constructor. Also the docs for `DefaultResolutionProvider` are outdated as they don't mention vulnerability resolutions. We probably shouldn't mention all types of resolutions there anyway.

Reply:
I'll make a refactoring in a separate PR.